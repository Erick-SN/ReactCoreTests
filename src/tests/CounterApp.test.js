import { shallow } from 'enzyme';
import CounterApp from '../components/CounterApp';
describe('Pruebas en el counter App', () => {
  test('Mostrar el componente <CounterApp /> de manera correcta', () => {
    const wraper = shallow(<CounterApp value={10} />);
    expect(wraper).toMatchSnapshot();
  });
  test('Mostrar un valor por defecto de 100', () => {
    const wraper = shallow(<CounterApp value={100} />);
    const counterText = wraper.find('h2').text().trim();
    expect(counterText).toBe('100');
  });
});
