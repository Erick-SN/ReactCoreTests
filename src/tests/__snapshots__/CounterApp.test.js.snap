// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el counter App Mostrar el componente <CounterApp /> de manera correcta 1`] = `
<Fragment>
  <div
    className="counter"
  >
    <h1>
      CounterApp
    </h1>
    <h2>
      10
    </h2>
    <button
      onClick={[Function]}
    >
       +1 
    </button>
    <button
      onClick={[Function]}
    >
       -1 
    </button>
    <button
      onClick={[Function]}
    >
      Reset
    </button>
  </div>
</Fragment>
`;
